test/specs/scale.time.tests.js

```diff
@@ -297,8 +297,8 @@ describe('Time scale tests', function() {
 		var xScale = chart.scales.xScale0;
 
 		// Counts down because the lines are drawn top to bottom
-		expect(xScale.ticks[0].value).toBe('Jan 2');
-		expect(xScale.ticks[1].value).toBe('May 8');
+		expect(xScale.ticks[0].value).toEqualOneOf(['Jan 1', 'Jan 2']);
```

Comment:
I'm not sure what the root cause is here, but my guess is it's timezone related? Changing the parser function didn't seem to fix it.

Reverting to the previous state seems like an improvement to me even if there's ultimately a better fix. I'd rather spend my time fixing issues that affect more people than invest more time in this